src/main/java/de/rub/nds/scanner/core/report/ScanReport.java

```diff
@@ -71,18 +54,18 @@ public abstract class ScanReport extends Observable {
     private final Map<TrackableValue, ExtractedValueContainer<?>> extractedValueContainerMap;
 
     private final List<GuidelineReport> guidelineReports;
-    private int score;
+    private Integer score;
     private ScoreReport scoreReport;
 
-    private final Set<ScannerProbe<?, ?>> executedProbes;
-    private final Set<ScannerProbe<?, ?>> unexecutedProbes;
+    @JsonIgnore private final Set<ScannerProbe<?, ?>> executedProbes;
+    @JsonIgnore private final Set<ScannerProbe<?, ?>> unexecutedProbes;
```

Comment:
I think the issue here is that we are storing the scanner probes and not the probe types (which we should I think)